### PR TITLE
Switch back to docker

### PR DESCRIPTION
**Pull request by rvem**
## Description
Problem: Podman turned out to be quite unreliable.

Solution: Switch back to docker, since we now have a dedicated agent
with access to the docker.

## Related issue(s)

None

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: docker/package/Dockerfile-ubuntu [hunk 1]
```diff
@@ -20,4 +20,4 @@ RUN opam install opam-bundle=0.4 --yes
 COPY docker/package/*.py /tezos-packaging/docker/package/
 COPY docker/package/defaults /tezos-packaging/docker/package/defaults
 COPY docker/package/scripts /tezos-packaging/docker/package/scripts
-ENTRYPOINT ["python3", "-m", "package.package_generator.py"]
+ENTRYPOINT ["python3", "-m", "package.package_generator"]
```
Comment (gromakovsky) [inline docker/package/Dockerfile-ubuntu:23]: Nice bugs.

**Review by gromakovsky — APPROVED** [review]
Looks fine, but I think someone from @serokell/operations should review as well.

File: docker/package/Dockerfile-ubuntu (hunk 1 above)
Comment (rvem) [reply docker/package/Dockerfile-ubuntu:23]: weird, that podman haven't caughted this error

**Review by balsoft — APPROVED** [review]
Superficially LGTM

**Pull request merged**